cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -33,7 +38,19 @@ def export_model_predictions(cloud_event: CloudEvent) -> dict:
         path.parts
     )
 
-    return _get_study_area_metadata(study_area_name)
+    try:
+        study_area_metadata = _get_study_area_metadata(study_area_name)
+    except ValueError as ve:
+        raise ve
+
+    try:
+        chunk_metadata = _get_chunk_metadata(study_area_metadata, chunk_id)
+    except ValueError as ve:
+        raise ve
```

Comment:
The same question about `try...except` as above.

Reply:
done